data-loader/src/v2/DataLoader.ts

```diff
@@ -117,6 +123,18 @@ export class DataLoader {
       } else {
         console.log(`WARNING: Failed to load entity with code ${entity.code}`);
       }
+      if (entity.type === 'Route') routes.add(entity.name.trim());
+      if (entity.type === 'Form') forms.add(entity.name.trim());
+      if (
+        entity.type === 'FormQualifier' &&
+        entity.name &&
+        entity.name.trim() &&
+        entity.name.trim().length < 20
```

Comment:
This filters out a good bunch of the vaccines

Reply:
Yeah, probably over invested in this code. Nice to have something we can use to re-set the data though, so think it's worth having some kind of loader working...